main.go

```diff
@@ -137,11 +159,20 @@ func main() {
 		case <-ticker:
 			fmt.Printf("%v runs so far, %v failures, over %s\n",
 				runs, fails, roundToSeconds(time.Since(startTime)))
+		case <-ctx.Done():
+			// A context timeout in this case is indicative of no failures
+			// being detected in the allotted duration.
+			if err := ctx.Err(); err != context.DeadlineExceeded {
+				return err
+			}
+			return nil
 		}
 	}
-	fmt.Printf("%v runs completed, %v failures, over %s\n",
-		runs, fails, roundToSeconds(time.Since(startTime)))
-	if fails > 0 {
```

Comment:
It'd be nice if we could preserve this final tally of the run/failure count. As it stands, `stress` runs that take less than 5s to complete don't print a tally at all.